collection-scripts/gather_odf_client

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Expect base collection path as an exported variable
+# If it is not defined, use PWD instead
+BASE_COLLECTION_PATH=${BASE_COLLECTION_PATH:-"$(pwd)"}
+
+
+STORAGE_CLIENT_NAMESPACE=$(oc get storageclients -A --no-headers | awk '{print $1}')
+CLIENT_OPERATOR_NAMESPACE=$(oc get subscriptions -A --no-headers | grep ocs-client-operator | awk '{print $1}')
+
+# collection path for OC commands
+mkdir -p "${BASE_COLLECTION_PATH}/odf-client/oc_output/"
+
+# Command List
+commands_get=()
+commands_get+=("pods")
+commands_get+=("deployments")
+commands_get+=("secrets")
+commands_get+=("configmaps")
+commands_get+=("cronjobs")
+commands_get+=("subscription")
+commands_get+=("storageclients")
+
+# YAML List
+oc_yamls=()
+oc_yamls+=("pods")
+oc_yamls+=("deployments")
+oc_yamls+=("secrets")
+oc_yamls+=("configmaps")
+oc_yamls+=("cronjobs")
+oc_yamls+=("subscription")
+oc_yamls+=("storageclients")
+
+# OC desc List
+commands_desc=()
+commands_desc+=("pods")
+commands_desc+=("deployments")
+commands_desc+=("secrets")
+commands_desc+=("configmaps")
+commands_desc+=("cronjobs")
+commands_desc+=("subscription")
+commands_desc+=("storageclients")
+
+
+for INSTALL_NAMESPACE in  $STORAGE_CLIENT_NAMESPACE $CLIENT_OPERATOR_NAMESPACE ; do
```

Comment:
we only need storageclient CRs from storage_client_namespace

Reply:
Storage client CRs are captured in must gather

Reply:
- "we only need storageclient CRs" --> we do not need anything else, like pods, deployments, secrets etc from namespaces that has storageclient CRs.
- if you are confused, refer https://github.com/red-hat-storage/odf-must-gather/pull/74#discussion_r1363109932 again